packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -75,11 +81,15 @@ export const getCheckboxStyles = (theme: GrafanaTheme2, invalid = false) => {
 
   return {
     wrapper: css`
-      display: inline-grid;
-      align-items: center;
-      column-gap: ${theme.spacing(labelPadding)};
-      position: relative;
-      vertical-align: middle;
+      display: inline-flex;
+      align-items: flex-start;
+      flex-direction: column;
+      & div:nth-child(1) {
```

Comment:
we should definitely not do this. this applies to any first child of **every** div that's a child of this element 😬 should be able to take a look after lunch

Reply:
😬 I was not sure about it, that was why I asked you for checking it